blocks/tag-title-block/features/tag-title/default.jsx

```diff
@@ -1,24 +1,19 @@
 import React from "react";
 import { useFusionContext } from "fusion:context";
-import "./tag-title.scss";
-import { PrimaryFont } from "@wpmedia/shared-styles";
+import { Heading, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-tag-title";
 
 export const TagTitleOutput = ({ data }) =>
 	// Check if tag exists
 	data && data.Payload && data.Payload[0] ? (
-		<>
-			<PrimaryFont as="h1" className="tag-name">
-				{data.Payload[0].name}
-			</PrimaryFont>
+		<Stack className={BLOCK_CLASS_NAME} direction="vertical">
+			<Heading>{data.Payload[0].name}</Heading>
```

Comment:
```suggestion
			{data.Payload[0].name ? <Heading>{data.Payload[0].name}</Heading> : null}
```

Update to check for heading presence before rendering. See my review for longer description.

Reply:
That makes sense! I thought about doing this, but wanted to keep my migration as "lean" as possible. However, I think with the addition of the chromatic changes, it totally makes sense to update the code here.